pkg/tsdb/prometheus/prometheus.go

```diff
@@ -70,6 +70,7 @@ func (s *Service) Init() error {
 
 func newInstanceSettings() datasource.InstanceFactoryFunc {
 	return func(settings backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
+		defaultHttpMethod := "POST"
```

Comment:
Probably need to import the "http" package with this change as well.

```suggestion
		defaultHttpMethod := http.MethodPost
```

Reply:
Changed in https://github.com/grafana/grafana/pull/38446/commits/18caa5b1fe25318a3c9e388a4e0d878ca189e055